### PR TITLE
Enable WebFrame method forwarding in sandboxed renderers

### LINKED ISSUE #9073: executeJavaScript does not work in sandbox mode
* Electron version: 1.6.4
* Operating system: macOS

### Expected behavior

executeJavaScript should return the result in callback or resolve/reject the promise
It works in not-sandbox mode

### Actual behavior

The callback is never called. The returned promise is never resolved

### How to reproduce

In did-finish-load:
```JS
webContents.executeJavaScript('123', false, res => {
   // never called
})
```
or
```JS
const res = await webContents.executeJavaScript('123')
// never resolved
```

Same if the js code is a promise or some other value.

### PR DESCRIPTION
**Pull request by nornagon**
Fixes #9073

### CODE REVIEW

**Comment by codebytere** [comment]
If you could add a test though @nornagon that would be great

**Comment by nornagon** [comment]
Done! Thanks for the review @codebytere :)

File: spec/api-web-contents-spec.js [hunk 1]
```diff
@@ -754,4 +754,23 @@ describe('webContents module', () => {
       })
     })
   })
+
+  describe('webframe messages in sandboxed contents', () => {
+    it('responds to executeJavaScript', (done) => {
+      w.destroy()
```
Comment (deepak1556) [inline spec/api-web-contents-spec.js]: This should not be needed, there is already a cleanup of open windows after each spec.

**Comment by nornagon** [comment]
Hey look it passed now 🎉

**Comment by nornagon** [comment]
See https://github.com/electron/electron/pull/13076.

**Pull request merged**